lib/board.rb

```diff
@@ -0,0 +1,74 @@
+class Board
+  attr_accessor :cells
+
+  #initiazies board as 9 element array with blank spaces
+  def initialize
+    @cells = Array.new(9," ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.